README.md

```diff
@@ -1,11 +1,13 @@
 # PyFlooder
 An [HTTP Flood](https://en.m.wikipedia.org/wiki/HTTP_Flood) Python script that could stop a normal website in 10s
 
-# How does it work ?
```

Comment:
Change it to
# How does it work & how its flow?
like what you did in the other PR

Reply:
Then I will merge